README.md

```diff
@@ -58,16 +58,15 @@ We previously built the component `common/components/todo-item.js` in [Part 1](h
 
 ```javascript
 // file: common/components/todo-item.js
-var React = require('react');
+import React from 'react';
 
-module.exports = React.createClass({
-  displayName: 'TodoItem',
-
-  getInitialState: function() {
-    return { done: this.props.done }
-  },
+export default class TodoItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { done: props.done };
+  }
```

Comment:
I'd much prefer to leave this as `React.createClass` - I'm not a fan of the `class`/`super` keywords in ES2015. Eric Elliott sums up my views pretty well [here](https://twitter.com/_ericelliott/status/573090480004591617).

Essentially: JS has an extremely powerful prototypal inheritance system. It can be used in a huge number of ways, most of which are *not* `class` style. For the times when a `class` style is used, it's usually going to end up in class hierarchy hell where you could have instead used a factory function to avoid the situation in the first place, and leave you with an easier to debug flat "hierarchy".

I understand that in this case the class isn't likely to be extended again, etc, but it's still encouraging the use of `class` / `super` which I'd prefer to avoid.